legion/legion/utils.py

```diff
@@ -450,7 +453,7 @@ def send_header_to_stderr(header, value):
     :type value: str
     :return: None
     """
-    message = 'X-Legion-%s:%s' % (header, value)
```

Comment:
I think we should put X-Legion- prefix to name of headers themselves. So, nobody need to look it up through the code.

Reply:
It has been fixed in #225